games-fps/chocolate-doom/chocolate_doom-3.0.0.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="Doom source port that accurately reproduces the experience of Doom"
+DESCRIPTION="Chocolate Doom is a source port of the game Doom, by id Software. \
+The purpose of Chocolate Doom is to be as compatible as possible with the \
+original DOS version of Doom."
+HOMEPAGE="https://www.chocolate-doom.org/"
+COPYRIGHT="Id Software Inc. 1993-1996
+	Simon Howard
+	James Haley
+	Samuel Villarreal"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://www.chocolate-doom.org/downloads/$portVersion/chocolate-doom-$portVersion.tar.gz"
+CHECKSUM_SHA256="73aea623930c7d18a7a778eea391e1ddfbe90ad1ac40a91b380afca4b0e1dab8"
+SOURCE_DIR="chocolate-doom-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	chocolate_doom$secondaryArchSuffix = $portVersion
+	cmd:chocolate_doom$secondaryArchSuffix = $portVersion
+	cmd:chocolate_doom_setup$secondaryArchSuffix = $portVersion
+	cmd:chocolate_heretic_setup$secondaryArchSuffix = $portVersion
+	cmd:chocolate_heretic$secondaryArchSuffix = $portVersion
+	cmd:chocolate_hexen_setup$secondaryArchSuffix = $portVersion
+	cmd:chocolate_hexen$secondaryArchSuffix = $portVersion
+	cmd:chocolate_server$secondaryArchSuffix = $portVersion
+	cmd:chocolate_setup$secondaryArchSuffix = $portVersion
+	cmd:chocolate_strife_setup$secondaryArchSuffix = $portVersion
+	cmd:chocolate_strife$secondaryArchSuffix = $portVersion
```

Comment:
Since this port don't work on x86_gcc2, you may want to consider using [`commandBinDir`][0] to strip the `_x86` suffix on x86_gcc2.

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#using-the-commandbindir-user-variable

Reply:
Yep, that's a very good idea.
BTW, it might be nice to include in this PR the removal of the old recipe: `chocolate_doom-2.3.0.recipe`.
(If 3.0.0 works fine then we can drop 2.3.0.)